custom_cards/custom_card_alarm_time/README.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Alarm Time Custom-card
+---
+
+<!-- markdownlint-disable MD046 -->
+
+# Custom-card "Alarm Time Card"
+
+The `card_alarm_time` you can control a input_datetime entity and a boolean.
+This card is useful for setting alarms (e.g. wakeup alarm), as you can modify the time and toggle the state of the alarm from one card.
+
+## Credits
+
+Author: benbur - 2023
+Version: 1.0.0
+
+This is built largely on top of the original Input_Datetime Card:
+Author: sildehoop - 2022
+Version: 1.0.0
+
+## Changelog
+
+<details>
+<summary>1.0.0</summary>
+Initial Release
+</details>
+
```

Comment:
Missing table with configurable variables and default values

Reply:
Good point, i'd forgotten that.